pytest_watch/tests/test_command.py

```diff
@@ -0,0 +1,437 @@
+import sys
```

Comment:
(Same comment in this file about using functions over class-based tests.)

Also `assert x != y` is much more readable than `self.assertFalse(condition)`, etc. Pytest does an amazing job of inspecting values when things go wrong, so I'm all for the boost in readability using raw `assert`.

Again, if you'd rather get this in now, that's fine. I'll probably go back and switch it later on if that's the case.

Reply:
Perfect. I agree with readability os simple assert. I will adopt this for general assertions.

I am in doubt about self.assertListEqual and self.assertDictEqual calls. I think it is better to use unittest helpers in this case. What do you think?

Thanks

Reply:
Well given that [pytest advocates the use of `assert`](https://docs.pytest.org/en/latest/assert.html) over `self.assert`, and this is a pytest-based project, I think using vanilla `assert` is appropriate. Here's the quote directly from the [Features list](https://docs.pytest.org/en/latest/#id1): "Detailed info on failing assert statements (no need to remember `self.assert*` names)"

FWIW readability was one of the things that initially attracted me to pytest to begin with.

Reply:
Done!